### ISSUE TITLE
Translate into other languages

### ISSUE THREAD

**Issue by monsterooo** [ralfbiedert/cheats.rs#167]
this is very great project. Are there any plans to translate into other languages now? such as Chinese, huge thanks。

**Comment by ralfbiedert** [comment]
I don't have plans to translate the site myself. There are some translation efforts though:

- An older Chinese one I don't endorse, as it claims it is hosted by me (it isn't)
- A Korean one is in the works #154

Feel free to do you own translation, or get in touch with one of the existing ones. From my side there aren't any special requirements apart from what's written in the "Legal" section. Ping if anything is unclear.

**Comment by jhq223** [comment]
Hi @ralfbiedert ,

Following up on the discussion in this issue, I'm excited to share that I've completed a new, up-to-date Chinese (Simplified) translation of cheats.rs!

I was very careful to follow the guidelines in your legal section, especially regarding attribution and the "Operator" details.

*   **Live Chinese Version:** [https://cheats-rs-zh.pages.dev/](https://cheats-rs-zh.pages.dev/)
*   **GitHub Repository (Fork):** [https://github.com/jhq223/cheats.rs-zh](https://github.com/jhq223/cheats.rs-zh)

The main translation work was assisted by Google's Gemini, and I handled the proofreading, technical setup, and will be responsible for its long-term maintenance to keep it in sync with your original project.

I hope this will be a valuable resource for the growing Chinese-speaking Rust community.

Thank you again for creating such a fantastic resource and for your open approach to community translations!
